examples/ml/housing-prices/regression.json

```diff
@@ -0,0 +1,25 @@
+{
+    "pipeline": {
+        "name": "regression"
+    },
+    "description": "A pipeline that trains produces a regression model for housing prices.",
+    "input": {
+        "pfs": {
+            "glob": "/*",
```

Comment:
Just to double check ourselves, do we want `/*` here or `/`?

How would your code handle multiple files added instead of the just 1 that is overwritten? Looks like it would create a new output for each input file, right?

Reply:
It's a good question. I had originally thought that each new file would be a different regression dataset (i.e. Boston Housing and separately California Housing). We could bake this in as a separate branch for each dataset as well though. If this example grows, the we would want to consider the dataset structure in more detail, because we would potentially want a dedicated testing set that doesn't change with new data (so models could be comparable). Do you this example would go more towards multiple datasets or growing a single dataset?